Classes/Projection/Doctrine/DoctrineProjectionPersistenceManager.php

```diff
@@ -33,10 +34,30 @@ class DoctrineProjectionPersistenceManager
     protected $systemLogger;
 
     /**
-     * @Flow\Inject
-     * @var EntityManager
+     * @var DoctrineEntityManager
      */
-    protected $entityManager;
+    private $entityManager;
+
+    /**
+     * @param DoctrineObjectManager $entityManager
+     * @return void
+     */
+    public function injectEntityManager(DoctrineObjectManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
```

Comment:
This change is also not directly related, but it allows for correct type hinting